### DISCUSSION TITLE
PaddleLiteV2.12 remove XShape OutputTensor for SqueezeOp.

### DISCUSSION THREAD

**Discussion by DarrenZhangug** [PaddlePaddle/PaddleOCR#14262] [Q&A]
环境rk3568
预测库：paddle-lite V2.12、paddleOCR V2.9
训练模型：ch_PP-OCRv4_rec_server_train
配置文件：ch_PP-OCRv4_rec_hgnet.yml
运行命令：./paddle_lite_ocr rec ch_ppocr_mobile_v2.0_rec_slim_opt.nb arm8 INT8 10 1 vis.jpg  ppocr_keys_v1.txt config.txt

以下是运行结果：
`[I 11/21 19:41:16.950 ...addle-Lite-2.12/lite/core/device_info.cc:1275 Setup] ARM multiprocessors name: HARDWARE        : ROCKCHIP RK3568 EVB1 DDR4 V10 BOARD

[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1276 Setup] ARM multiprocessors number: 4
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1278 Setup] ARM multiprocessors ID: 0, max freq: 1992, min freq: 1992, cluster ID: 0, CPU ARCH: A55
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1278 Setup] ARM multiprocessors ID: 1, max freq: 1992, min freq: 1992, cluster ID: 0, CPU ARCH: A55
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1278 Setup] ARM multiprocessors ID: 2, max freq: 1992, min freq: 1992, cluster ID: 0, CPU ARCH: A55
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1278 Setup] ARM multiprocessors ID: 3, max freq: 1992, min freq: 1992, cluster ID: 0, CPU ARCH: A55
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1284 Setup] L1 DataCache size is: 
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1286 Setup] 32 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1286 Setup] 32 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1286 Setup] 32 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1286 Setup] 32 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1288 Setup] L2 Cache size is: 
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1290 Setup] 512 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1290 Setup] 512 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1290 Setup] 512 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1290 Setup] 512 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1292 Setup] L3 Cache size is: 
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1294 Setup] 0 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1294 Setup] 0 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1294 Setup] 0 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1294 Setup] 0 KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1296 Setup] Total memory: 1998392KB
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1297 Setup] SVE2 support: 0
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1298 Setup] SVE2 f32mm support: 0
[I 11/21 19:41:16.951 ...addle-Lite-2.12/lite/core/device_info.cc:1299 Setup] SVE2 i8mm support: 0
[I 11/21 19:41:17.161 ...e-Lite-2.12/lite/operators/squeeze_op.cc:124 AttachImpl] PaddleLiteV2.12 remove XShape OutputTensor for SqueezeOp.`
运行结果图片如下：
![err](https://github.com/user-attachments/assets/d4d98ed0-23e5-4447-b1b4-6a29731cddf9)

看报错应该时从paddle-lite V2.12起XShape 就被移除了的原因。我尝试替换成paddle-lite V2.13 V2.14这些预测库，结果都行，V2.10，2.10的库应为缺少paddle_use_kernels.h编译不过去。所以想在这里请教一下怎样解决这个问题，万分感谢！！

**Comment by GreatV** [comment]
paddle-lite 相关的问题建议在paddle-lite仓库提交issue，获取更专业的回复。

**Reply by DarrenZhangug** [reply]
好的，谢谢！
